swagger/specification/applications/resource-manager/Applications.Core/preview/2023-10-01-preview/openapi.json

```diff
@@ -4966,6 +5011,16 @@
         }
       }
     },
+    "Secret": {
+      "type": "object",
+      "description": "Specifies the secret details of type personal access token for each different git platforms",
+      "properties": {
+        "secretStore": {
```

Comment:
From the design I thought this was expected to be `secret` and not `secretStore` https://github.com/radius-project/design-notes/blob/3644b754152edc97e641f537b20cf3d87a386c43/recipe/2024-01-support-private-terraform-repository.md?plain=1#L106. Did we change that or am I'm interpreting this generated schema incorrectly?

Reply:
I think it should be `ApplicationProperties` instead of `EnvironmentProperties`https://github.com/radius-project/radius/blob/0a1a8680dace317f8bd25bdde2b503f895766d2d/typespec/Applications.Core/applications.tsp#L145
Maybe a typo while creating this file. Will create a fix for this in different PR.
Good Catch!!

Reply:
i changed it to secret.